list_lookup_test.go

```diff
@@ -0,0 +1,358 @@
+				client: mockHTTPClient(func(req *http.Request) *http.Response {
+					body := `{
+						"data": [
+							{
+								"follower_count": 0,
+								"id": "1451305624956858369",
+								"name": "Test List",
+								"owner_id": "2244994945"
+							}
+						],
+						"includes": {
+							"users": [
+								{
+									"username": "TwitterDev",
+									"id": "2244994945",
+									"created_at": "2013-12-14T04:35:55.000Z",
+									"name": "Twitter Dev"
+								}
+							]
+						},
+						"meta": {
+							"result_count": 1
+						}
+					}`
+					return &http.Response{
+						StatusCode: http.StatusOK,
+						Body:       io.NopCloser(strings.NewReader(body)),
+					}
+				}),
+				id: "2244994945",
+				opt: []*gotwtr.ListLookUpOption{
+					{
+						UserFields: []gotwtr.UserField{
+							gotwtr.UserFieldUserName,
+							gotwtr.UserFieldID,
+							gotwtr.UserFieldCreatedAt,
+							gotwtr.UserFieldName,
+						},
+					},
+				},
+			},
+			want: &gotwtr.OwnedListsLookUpByIDResponse{
+				Lists: []*gotwtr.List{
+					{
+						FollowerCount: 0,
+						ID:            "1451305624956858369",
+						Name:          "Test List",
+						OwnerID:       "2244994945",
+					},
+				},
+				Includes: &gotwtr.ListIncludes{
+					Users: []*gotwtr.User{
+						{
+							UserName:  "TwitterDev",
+							ID:        "2244994945",
+							CreatedAt: "2013-12-14T04:35:55.000Z",
+							Name:      "Twitter Dev",
+						},
+					},
+				},
+				Meta: &gotwtr.ListMeta{
+					ResultCount: 1,
+				},
+				Errors: nil,
+			},
+			wantErr: false,
+		},
+		{
+			name: "404 not found",
+			args: args{
+				ctx: context.Background(),
+				client: mockHTTPClient(func(req *http.Request) *http.Response {
+					body := `{
+						"errors":[
+							{
+								"parameters":{
+									"id":[
+										"111111111111111111111111"
+									]
+								},
+								"message":"The id query parameter value [111111111111111111111111] is not valid"
+							}
+						],
+						"title":"Invalid Request",
+						"detail":"One or more parameters to your request was invalid.",
+						"type":"https://api.twitter.com/2/problems/invalid-request"
+					}`
+					return &http.Response{
+						StatusCode: http.StatusNotFound,
+						Body:       io.NopCloser(strings.NewReader(body)),
+					}
+				}),
+				id:  "111111111111111111111111",
+				opt: []*gotwtr.ListLookUpOption{},
+			},
+			want: &gotwtr.OwnedListsLookUpByIDResponse{
+				Lists: nil,
+				Errors: []*gotwtr.APIResponseError{
+					{
+						Parameters: gotwtr.Parameter{
+							ID: []string{"111111111111111111111111"},
+						},
+						Message: "The id query parameter value [111111111111111111111111] is not valid",
+					},
+				},
+				Title:  "Invalid Request",
+				Detail: "One or more parameters to your request was invalid.",
+				Type:   "https://api.twitter.com/2/problems/invalid-request",
+			},
+			wantErr: true,
+		},
+	}
+	for i, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
@sivchari 
Isn't this it?

Reply:
@mmmommm
you can refer [this discussion](https://github.com/sivchari/gotwtr/pull/44#discussion_r753326996)

Reply:
@kazdevl 
ty :) I fixed